params_test.go

```diff
@@ -140,3 +140,31 @@ func TestParseBrowsersParam_MultiBrowserParam_AllInvalid(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Empty(t, browsers)
 }
+
+func TestParseMaxCountParam(t *testing.T) {
+	r := httptest.NewRequest("GET", "http://wpt.fyi/?max-count=2", nil)
+	count, err := ParseMaxCountParam(r)
+	assert.Nil(t, err)
+	assert.Equal(t, 2, count)
+}
+
+func TestParseMaxCountParam_Missing(t *testing.T) {
+	r := httptest.NewRequest("GET", "http://wpt.fyi/", nil)
+	count, err := ParseMaxCountParam(r)
+	assert.Nil(t, err)
+	assert.Equal(t, MaxCountDefaultValue, count)
+}
+
+func TestParseMaxCountParam_TooSmall(t *testing.T) {
+	r := httptest.NewRequest("GET", "http://wpt.fyi/?max-count=0", nil)
```

Comment:
Also try max-count=-1 or max-count=1e8 to see that the "not a non-negative integer" error handling is right?

Reply:
Done.